cetlvast/suites/docs/examples/example_07_polymorphic_alloc_deleter.cpp

```diff
@@ -9,43 +9,98 @@
 /// cSpell: words emplacer
 
 #include "cetl/pf17/cetlpf.hpp"
+#include "cetl/pmr/interface_ptr.hpp"
 #include "cetl/pmr/memory.hpp"
 
 #include <unordered_map>
 #include <iostream>
-#include <string.h>
 
 #include <gtest/gtest.h>
 
+class INameable
```

Comment:
In the end of this file I introduce new "example_usage_2" test case - this is where I demonstrate the "interface"-part feature of new `cetl::pmr::InterfacePtr`. So, I need couple interfaces for the `MyObject` final class.

Reply:
`INameable` implies that you can give it a name or use `name` as a verb, no? If it has a name that can't be changed then shouldn't it be `INamed`? Then the second one would be `IDescribed` (or you'd need to change `description()` into `describe()`). I think there is no problem with `IIdentifiable` because one could argue that it uses `id` as a verb.

Normally I would consider it a nitpick but this is a doc example where names matter.

Reply:
I believe there is no such notion that "xxx-able" is necessarily has some kind of "setter" logic. IMO all it says that there is a "xxx" verb which can be applied to an entity. In your example with "id" it looks like it is just shortcut for "identify" verb.

Also, in my experience past tense suffix (-ed) is commonly in use to connote "returning a new instance of something transformed", like const `arr.sorted()` vs in-place `arr.sort()` mutation.